src/deploy-cromwell-on-azure/scripts/helm/templates/tes-deployment.yaml

```diff
@@ -66,8 +66,6 @@ spec:
               value: {{ .Values.config.disableBatchScheduling | quote }}
             - name: DockerInDockerImageName
               value: {{ .Values.config.dockerInDockerImageName }}
-            - name: UseLegacyBatchImplementationWithAutopools
```

Comment:
This is still used. It shouldn't be removed.

Reply:
Probably a merge conflict, but it was in the tes-deployment.yaml file twice.